plugins/org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/model/GeneratedJavaFileAccess.xtend

```diff
@@ -25,9 +26,17 @@ class GeneratedJavaFileAccess extends JavaFileAccess {
 		if (!markedAsGenerated)
 			throw new IllegalArgumentException("It's always generated");
 	}
-
+	
+	def setTypeComment(StringConcatenationClient javaContent) {
```

Comment:
the setter stem's from 'TextFileAccess' and declaring it explicitly 'void' does not break anything.
Personally I'm not a fan of void, but rather would return the '...FileAccess' instance itself, like it is done in ``GuiceModuleAccess.BindingFactory``. But that may be a bit overdressed here.
Anyway, returning the 'CharSequence' doesn't make sense to me.